### PR TITLE
Make textarea and output full screen

### PR DESCRIPTION
**Pull request by s-katte**
### Here is the screenshot of changes made.
image

### CODE REVIEW

File: templates/asciidoc.html [hunk 1]
```diff
@@ -44,11 +44,11 @@
                     <div class="ui grid">
                         <div class="eight wide column">
                             <div class="ui tiny form field">
-                                <textarea rows="2" id="textdata" class="monotext" onkeyup="autoconv(); dispatch();"></textarea>
+                                <textarea style="height: 81vh;" rows="2" id="textdata" class="monotext" onkeyup="autoconv(); dispatch();"></textarea>
```
Comment (t0xic0der) [inline templates/asciidoc.html:47]: 81vh is unfortunately not uniform for all the resolutions.

This is how it looks like in 768p.
image

This is how it looks like in 720p.
image

This is how it looks like in 900p.
image

And finally this is how it looks like in 1080p.
image

As you can see, 1920x1080 is the only resolution where it looks the way it is supposed to and hence leaves away the general scrollbar. For the remaining resolutions, eg. 1600x900, 1280x720 and 1366x768 which are in fact standard resolutions for devices - it tries to occupy more space than what is available on the viewport - thus enabling the scrollbar.

File: templates/asciidoc.html [hunk 2]
```diff
@@ -44,11 +44,11 @@
...
                             </div>
                         </div>
                         <div class="eight wide column" style="border-width: 2px; border-radius: 2px;">
-                            <div class="ui form textbase" style="border: 1px solid #dedede; border-radius: 5px; height: 100%; padding: 1%;" id="otptdata"></div>
+                            <div class="ui form textbase" style="overflow-y: scroll; height:81vh; border: 1px solid #dedede; border-radius: 5px; padding: 1%;" id="otptdata"></div>
```
Comment (t0xic0der) [inline templates/asciidoc.html:51]: Has the same repercussions as mentioned in the former review comment.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Try to ensure that only the entirety of the available view space is claimed and not more. With resolutions like 1440p or 2160p (2560x1440 and 3840x2160 respectively), your code might end up occupying less than the complete view space.

File: templates/asciidoc.html (hunk 2 above)
Comment (s-katte) [reply templates/asciidoc.html:51]: Sure, will do it.

**Comment by s-katte** [comment]
Have a look at it now @t0xic0der

**Pull request closed without merging**